fprettify/__init__.py

```diff
@@ -2057,13 +2080,14 @@ def build_ws_dict(args):
                 sys.exit(1)
         else:
             if not os.path.exists(directory):
-                sys.stderr.write("directory " + directory +
+                sys.stderr.write("file " + directory +
                                  " does not exist!\n")
                 sys.exit(1)
-            if not os.path.isfile(directory) and directory != '-' and not args.recursive:
-                sys.stderr.write("file " + directory + " does not exist!\n")
+            if os.path.isdir(directory) and not args.recursive:
```

Comment:
I think this is a bit clearer. Also the `directory != '-'` condition was redundant here.